search/quality_eval.go

```diff
@@ -169,6 +172,7 @@ const (
 	FILTER_NAME_TOPIC        = "topic"
 	FILTER_NAME_CONTENT_TYPE = "contentType"
 	PREFIX_LATEST            = "[latest]"
+	BLOG_OR_TWEET_MARK       = "blog" //  TBD consider to change the value
```

Comment:
Change this to blog_or_tweet   // Update the recall set that you are committing.
When we commit this we also update the recall set in spreadsheet.

Reply:
changed, recall updated in source code and spreadsheet